### PR TITLE
Add rake db:prepare entry to the CHANGELOG.md

### PR DESCRIPTION
**Pull request by robertomiranda**
Adding a missing entry into the `CHANGELOG.md` file.

ref https://github.com/rails/rails/pull/35768#issuecomment-479396632

### CODE REVIEW

**Review by simi — APPROVED** [review]

File: activerecord/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+*   Add `rake db:prepare`  runs setup if database does not exist, or runs migrations if it does
```
Comment (kamipo) [inline activerecord/CHANGELOG.md]: Don't need double spaces.

```suggestion
*   Add `rake db:prepare` runs setup if database does not exist, or runs migrations if it does.
```

Comment (simi) [inline activerecord/CHANGELOG.md]: nitpick: _I think it is common to add entry as a sentence, to make this sentence I think it should end with **.**_ symbol.

**Comment by kamipo** [comment]
Thanks. In the future, you can add `[ci skip]` to your commit message for documentation-only changes to skip the CI build.

**Comment by robertomiranda** [comment]
oh thanks, didn't know that @kamipo

File: activerecord/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+*   Add `rake db:prepare` runs setup if database does not exist, or runs migrations if it does.
```
Comment (sharang-d) [inline activerecord/CHANGELOG.md]: We can replace 'runs' with 'which runs'.

**Comment by kaspth** [comment]
Reformatted this in 9864f5e3d6 to fit the writing style of other entries — also switches to `rails` instead of `rake`. @robertomiranda thanks for the command!

**Comment by robertomiranda** [comment]
thanks @kaspth

**Pull request merged**